test/integration/targets/meraki_network/tasks/main.yml

```diff
@@ -161,10 +163,69 @@
       state: present
       org_name: '{{test_org_name}}'
       net_name: IntTestNetworkCombined
-      disable_my_meraki: no
+      enable_my_meraki: yes
     delegate_to: localhost
     register: enable_meraki_com
-    
+
+  - name: Disable my.meraki.com for next test
+    meraki_network:
+      auth_key: '{{ auth_key }}'
+      state: present
+      org_name: '{{test_org_name}}'
+      net_name: IntTestNetworkCombined
+      enable_my_meraki: no
+    delegate_to: localhost
```

Comment:
`delegate_to` could be set once at `block` level for all tasks in this file:
```
- delegate_to: localhost
  block:
    - name: Create network without type
[...]
```

Reply:
Updated.